Changed download page to add wiki link for older ubuntu users

Helios747 opened the pull request:
Also added wiki link for other distro users to compile dolphin.

I find it very unfortunate that Ubuntu 14.04 LTS users have no immediate direction for installing Dolphin. I know we're not technically supporting them but I feel like there should be an easy way for LTS users to still receive support. Sergio-br2 maintains the PPA and it has been kept up to date for awhile now. It also seems to be kept more up to date then Glennric's PPA. He has also said that he intends on staying with LTS releases.

sergiobenrocha2 commented:
I think you can use 14.04 instead 14.10. Utopic will lose its support soon (next month) --> https://wiki.ubuntu.com/Releases

delroth commented on dolweb/downloads/templates/downloads-index.html:
```diff
@@ -87,6 +87,14 @@
 </table>
 </div>
 
+<div id="Other-Linux">
```
Not translatable.

delroth replied:
Can you rename the id to "other-linux" for consistency with other ids on that page?

aserna3 commented:
Ohhh, do I need to add a {% trans % } tag around the <h1> and <p> text? I was wondering what those tags did.

Zopieux commented:
Yes, please use `{% trans %}` tags for short texts and `{% blocktrans %}` blocks for longer ones (or ones with variables). Also, please use `<br>` scarcely, you can just create new paragraphs.

```html+django
<h1>{% trans "Older Ubuntu (< 15.04) and other Linux distros" %}</h1>
<p>{% blocktrans %}Ubuntu 14.10 and older users can install a PPA
for development and stable versions of Dolphin here:
<a href="https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu">Installing Dolphin</a>
{% endblocktrans %}</p>
{# and so on #}
```

delroth commented on dolweb/downloads/templates/downloads-index.html:
```diff
@@ -87,6 +87,15 @@
+<div id="Other-Linux">
+    <h1>{% trans "Older Ubuntu (< 15.04) and other Linux distros" %}</h1>
+    <p>{% blocktrans %}Ubuntu 14.10 and older users can install a PPA 
+    for development and stable versions of Dolphin here: <a href=https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu>Installing Dolphin</a>
```
Missing "" around link hrefs.

delroth commented on dolweb/downloads/templates/downloads-index.html:
```diff
@@ -87,6 +87,15 @@
+    <p>{% blocktrans %}Ubuntu 14.10 and older users can install a PPA 
+    for development and stable versions of Dolphin here: <a href=https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu>Installing Dolphin</a>
+    {% endblocktrans %}</p>
+    <p>{% blocktrans %}Other Linux distro users can look here to compile Dolphin: <a href=https://wiki.dolphin-emu.org/index.php?title=Building_Dolphin_on_Linux>Building Dolphin on Linux</a>
```
I'm not a native english speaker, but "Other Linux distro users" sounds weird to me. First of all, s/distro/distributions/, no need to abbreviate in this context. Second of all, I would turn it around like "Users of other Linux distributions".

But again, ignore if I'm wrong here.

Helios747 committed:
Changed download page to add wiki link for older ubuntu users

Also added wiki link for other distro users to compile dolphin

Added translation tags and removed unneeded line break tags

Fixed quotes around links and reworded text (take two)

fixed casing on div tag for consistency

delroth commented:
@JMC47 please review wording and other stuff.
